app/build.gradle.kts

```
@@ -1,45 +1,53 @@
 // NOTE: this project uses Gradle Kotlin DSL. More common build.gradle instructions can be found in
 // the main README.
 plugins {
-  id("com.android.application")
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("org.jetbrains.kotlin.plugin.compose")
+    id("org.jetbrains.compose")
+    id("com.google.gms.google-services")
+    id("kotlin-kapt")                     // ← kapt drives annotation processors
 }
 
 android {
+    namespace  = "com.firebase.uidemo"
     compileSdk = Config.SdkVersions.compile
 
-    namespace = "com.firebase.uidemo"
-
     defaultConfig {
-        minSdk = Config.SdkVersions.min
-        targetSdk = Config.SdkVersions.target
-
-        versionName = Config.version
-        versionCode = 1
-
+        minSdk       = Config.SdkVersions.min
+        targetSdk    = Config.SdkVersions.target
+        versionName  = Config.version
+        versionCode  = 1
```

Let's get rid of all the extra whitespaces (across all files) - I don't think it's common to use this formatting in Kotlin (at least I've never seen before)

```suggestion
        minSdk = Config.SdkVersions.min
        targetSdk = Config.SdkVersions.target
        versionName = Config.version
        versionCode = 1
```